subo/root.go

```diff
@@ -26,6 +26,9 @@ Explore the available commands by running 'subo --help'`,
 		Use:   "create",
 		Short: "create a runnable or project",
 		Long:  `create a new Atmo project or WebAssembly runnable`,
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Should this RunE be added to other top-level commands like `compute`, `create`, and `deploy` below?

Reply:
@samderlust are you adding it to the others too?

Reply:
yeah I can do that